src/core/aws-net-websocket-protocol-rfc6455.adb

```diff
@@ -122,6 +180,18 @@ package body AWS.Net.WebSocket.Protocol.RFC6455 is
       Protocol.Close_Sent := True;
    end Close;
 
+   ------------------------
+   -- Create_Random_Mask --
+   ------------------------
+
+   function Create_Random_Mask return Masking_Key is
+      Int : constant AWS.Utils.Random_Integer := AWS.Utils.Random;
+      Arr : Masking_Key with Import;
```

Comment:
No, it is to avoid any implicit initialization. Always recommended when you change the Address clause, since otherwise I might end up overwriting Int.

Reply:
Note that the `'Address` clause can be turned into an aspect itself. Maybe the following would be clearer:
```ada
Arr : Masking_Key with Import, Address => Int'Address;
```

Reply:
Yes, sounds even better!